Hazelcast.Net/Hazelcast.Config/ProxyFactoryConfig.cs

```diff
@@ -14,6 +14,9 @@
 
 namespace Hazelcast.Config
 {
+    /// <summary>
+    /// This class is related to SPI. To register custom services to client.
```

Comment:
/// This class is related to SPI; it is used to register custom services to client.